README.md

```diff
@@ -1,10 +1,28 @@
+| Documentation                  | Code Samples      | Library Reference | Support | Feedback|
+|-------------------------------|---------------------------|-------------------|---------|----------|
+| [MSAL iOS and macOS documentation](https://learn.microsoft.com/en-us/entra/msal/objc/) | &#8226;  [Microsoft Entra ID (workforce samples)](https://learn.microsoft.com/en-us/entra/identity-platform/sample-v2-code?tabs=framework#ios)<br/>&#8226; [Microsoft Entra External ID (customer samples)](https://learn.microsoft.com/en-us/entra/external-id/customers/samples-ciam-all?tabs=apptype#mobile)          | [ SDK reference](https://azuread.github.io/microsoft-authentication-library-for-objc/)             | [Get support](README.md#community-help-and-support) | [Feedback](https://forms.office.com/r/xuBV0CzEih) |
+
+
+The Microsoft Authentication Library (MSAL) for iOS and macOS is an Auth SDK that can be used to seamlessly integrate authentication into your apps using industry standard OAuth2 and OpenID Connect. It allows you to sign in users or apps with Microsoft identities. These identities include Microsoft Entra ID work and school accounts, personal microsoft accounts, social accounts, customer accounts, and more. Using MSAL for iOS and macOS, you can acquire security tokens from the Microsoft identity platform to authenticate users and access secure web APIs for their applications.
+
+MSAL for iOS and macOS supports multiple authentication scenarios, such as single sign-on (SSO), conditional access, and brokered authentication. The library also provides native authentication APIs that allow applications to implement a native experience with end-to-end customizable flows in their mobile applications. 
+
+## Get started
+
+To use MSAL iOS and macOS in your application, you need to register your application in the Microsoft Entra Admin center and configure your project. Since the SDK supports both browser-delegated and native authentication experiences, follow the steps in the one of these quickstarts based on your scenario.
+
+* For browser-delegated authentication scenarios, refer to the quickstart, [Sign in users and call Microsoft Graph from an iOS or macOS app](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-mobile-app-ios-sign-in).
+
+* For native authentication scenarios, refer to the Microsoft Entra External ID sample guide, [Run iOS sample app](https://review.learn.microsoft.com/en-us/entra/external-id/customers/how-to-run-native-authentication-sample-ios-app?branch=release-native-auth-public-preview).
+
+## Migrate from ADAL Objective-C
+
+The Azure Active Directory Authentication Library (ADAL) for Objective-C was deprecated on June 2023. Follow the [ADAL to MSAL migration guide for iOS and macOS](https://learn.microsoft.com/en-us/entra/msal/objc/migrate-objc-adal-msal) to avoid putting your app's security at risk.
```

Comment:
```suggestion
The Azure Active Directory Authentication Library (ADAL) for Objective-C has been deprecated effective June 2023. Follow the [ADAL to MSAL migration guide for iOS and macOS](https://learn.microsoft.com/en-us/entra/msal/objc/migrate-objc-adal-msal) to avoid putting your app's security at risk.
```